package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "moviehub",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "jest --env=jsdom",
+    "start": "webpack serve --open",
+    "build": "webpack",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/milen-ale/movieHub.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/milen-ale/movieHub/issues"
+  },
+  "homepage": "https://github.com/milen-ale/movieHub#readme",
+  "devDependencies": {
+    "@babel/core": "^7.18.10",
+    "@babel/preset-env": "^7.18.10",
+    "babel-jest": "^28.1.3",
+    "css-loader": "^6.7.1",
+    "gh-pages": "^4.0.0",
+    "html-webpack-plugin": "^5.5.0",
+    "jest": "^28.1.3",
+    "style-loader": "^3.3.1",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.10.0"
+  },
+  "dependencies": {
```

Comment:
- [ ] I kindly suggest you add `cross-fetch` as a dependency because the project required I installed `cross-fetch` and did not work until I installed it. you can add `cross-fetch` by running `npm i cross-fetch` on the command line.

## EXAMPLE
```
  "dependencies": {
    "cross-fetch": "^3.1.5",
  }
}
```

<img width="763" alt="scr2" src="https://user-images.githubusercontent.com/52934128/184507943-0b6b1b8c-7f1b-4b23-8fd4-0441ecfc3129.PNG">